crates/rspack_plugin_css/src/visitors/mod.rs

```diff
@@ -42,19 +48,41 @@ struct Analyzer<'a> {
   diagnostics: &'a mut Vec<Box<dyn Diagnostic + Send + Sync>>,
   nearest_at_import_span: Option<Span>,
   url_function_span: Option<Span>,
+  source_code: &'a str,
+  filename: &'a str,
   // in_support_contdition: bool,
 }
 
 fn replace_module_request_prefix(
   specifier: String,
   diagnostics: &mut Vec<Box<dyn Diagnostic + Send + Sync>>,
+  source_code: &str,
+  filename: &str,
+  span: Span,
 ) -> String {
   if IS_MODULE_REQUEST.is_match(&specifier) {
```

Comment:
nitpick: `IS_MODULE_REQUEST.find(...)` looks enough here. Then we can remove `Regex::new("~")`.